plugins/org/src/components/Cards/Group/MembersList/MembersListCard.tsx

```diff
@@ -144,23 +151,35 @@ export const MembersListCard = (_props: {
     return <ResponseErrorPanel error={error} />;
   }
 
+  const nbPages = Math.ceil((members?.length || 0) / pageSize);
+  const paginationLabel = nbPages < 2 ? '' : `, page ${page} of ${nbPages}`;
+
   return (
     <Grid item>
       <InfoCard
```

Comment:
I think if you pass the `Pagination` a prop here under `actions` it should get rendered outside of the content and make the styling look a little nicer.

```tsx
const pagination = (<Pagination
          count={nbPages}
          page={page}
          onChange={pageChange}
          showFirstButton
          showLastButton
        />
);

<InfoCard actions={pagination} />
```

Reply:
Just commit this change, It is a little bit nicer and more :

![screenshot-localhost_7000-2021 05 10-14_00_23](https://user-images.githubusercontent.com/24318548/117656284-75892680-b198-11eb-907a-918a0c4bb2cd.png)